runconfig/parse.go

```diff
@@ -70,6 +72,8 @@ func Parse(cmd *flag.FlagSet, args []string) (*Config, *HostConfig, *flag.FlagSe
 	cmd.Var(&flVolumes, []string{"v", "-volume"}, "Bind mount a volume")
 	cmd.Var(&flLinks, []string{"#link", "-link"}, "Add link to another container")
 	cmd.Var(&flDevices, []string{"-device"}, "Add a host device to the container")
+	cmd.Var(&flLabels, []string{"l", "-label"}, "Set meta data on a container")
```

Comment:
Saying that, I guess environment variables also don't mention formatting.

This help document could link off to the online documentation at the bottom for more information. That'd be pretty cool. I'll open a PR for that...